src/components/Home/Home.tsx

```diff
@@ -1,4 +1,15 @@
 import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { LaunchFilter } from '../LaunchFilter/LaunchFilter';
 import { Slider } from '../Slider';
+import './Home.scss';
 
-export const Home = () => <Slider />;
+export const Home = () => (
+  <div className="home-container">
+    <Slider />
+    <LaunchFilter />
+    <div className="home-inner-container">
+      <Outlet />
+    </div>
```

Comment:
Why only Outlet in the inner container?

Reply:
It's an error. I inserted Slider component into inner container